src/policies/__id/peoplefinder.DELETE.api.users.__id.rego

```diff
@@ -0,0 +1,25 @@
+package peoplefinder.DELETE.api.users.__id
+
+import input.policy.path
+import input.user.applications.peoplefinder
```

Comment:
we're no longer using input.user.applications.peoplefinder - we should be using input.user.attributes